scripts/generate/constants.ts

```diff
@@ -1,3 +1,3 @@
 // corresponds to Extension::Model::Type.identifier
 // https://github.com/Shopify/shopify-app-cli/tree/master/lib/project_types/extension/models/types
-export const extensionTypes = ['PRODUCT_SUBSCRIPTION'];
+export const extensionTypes = ['PRODUCT_SUBSCRIPTION', 'DEFAULT'];
```

Comment:
I don't think we need to add `DEFAULT` as a type since it's not a real type

Reply:
My suggestion would be to add a constant for the default fallback template name and then pull that into the generate command

```ts
export const DEFAULT_TEMPLATE = 'DEFAULT';


...

import {DEFAULT_TEMPLATE} from './constants';

async function init({type, template: templateIdentifier}: InitConfig) {

  const const extensionType = String(type);
  const template = fs.existsSync(`${process.cwd()}/${extensionType}`) ? extensionType : DEFAULT_TEMPLATE;


```